LICENCE.md

```diff
@@ -1,10 +1,10 @@
 The MIT License (MIT)
 =====================
 
-Copyright (c) 2016 nodereport contributors
+Copyright (c) 2016 node-report contributors
 --------------------------------------------------
 
-*nodereport contributors listed at <https://github.com/nodejs/nodereport/blob/master/AUTHORS>*
+*node-report contributors listed at <https://github.com/nodejs/node-report/blob/master/AUTHORS>*
```

Comment:
If we're updating this file, https://github.com/nodejs/nodereport/blob/master/AUTHORS no longer exists (it was removed by #13).